### ISSUE TITLE
Audio glitches

### ISSUE THREAD

**Issue by Rabittelong** [tpunix/SAROO#106]
Buy my saroo from Timeharvest Store updated to firm v0.4 and most of the games start fine but then the sound glitches at random with buzzes and wrong sounds playing, I'm at Japanese VA0 Saturn the weird thing is not all the games are affected by this issue.

**Comment by tzmwx** [comment]
It may be that a clock circuit on the VA0\VA0.5\VA0.8 motherboard does not have shielding protection.It is an early motherboard design problem.

Might need to try the motherboard: 
Can you take apart a Sega Saturn yourself?
Are you proficient in using a soldering iron?

If you are interested, you can continue to leave a message

**Comment by Rabittelong** [comment]
> It may be that a clock circuit on the VA0\VA0.5\VA0.8 motherboard does not have shielding protection.It is an early motherboard design problem.
> 
> Might need to try the motherboard: Can you take apart a Sega Saturn yourself? Are you proficient in using a soldering iron?

Sorry no I don't have any skill soldering.

**Comment by tzmwx** [comment]
> > It may be that a clock circuit on the VA0\VA0.5\VA0.8 motherboard does not have shielding protection.It is an early motherboard design problem.
> > Might need to try the motherboard: Can you take apart a Sega Saturn yourself? Are you proficient in using a soldering iron?
> 
> Sorry no I don't have any soldering skill.

Ok
then wait for his test results.
https://github.com/tpunix/SAROO/issues/35

**Comment by TheUndertakerPoland** [comment]
I can recommend using tape on on the smal motherboard method. Took about 10 min and fully working. CD ROM not working but audio works from saroo and sound fx without glitches. Tested today.

**Comment by cpg94** [comment]
> It may be that a clock circuit on the VA0\VA0.5\VA0.8 motherboard does not have shielding protection.It is an early motherboard design problem.
> 
> Might need to try the motherboard: Can you take apart a Sega Saturn yourself? Are you proficient in using a soldering iron?

I have some soldering skills and have had the same issue with sound. What can be done to solve the issue with soldering?

**Comment by TheUndertakerPoland** [comment]
I don't think you have to soldiering. Use the trick with tape... if you need optical drive cove just 1 of the pins on the cdrom small motherboard. If drive is not necessary use big piece of tape and cover following pins

**Comment by czapa86** [comment]
You can try removing C47 cap but someone reported it didn't change anything.
https://github.com/tpunix/SAROO/issues/35#issuecomment-2119020749

**Comment by TheUndertakerPoland** [comment]
That was not working for me. No change

Wysłano z programu Outlook dla systemu Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by cpg94** [comment]
Yeah I did the tape trick on the CD drive pin in the end and the Saroo is working fine now with not problems. Just wondered if there was a solution that means I can still use my CD drive if needed.

**Comment by tzmwx** [comment]
> Yeah I did the tape trick on the CD drive pin in the end and the Saroo is working fine now with no problems. Just wondered if there was a solution that means I can still use my CD drive if needed.

If only the B7 PIN is processed, the CD drive will not be affected. You can still use CD

**Comment by jerbearwright** [comment]
I can confirm this works also. I was having the same issues with a MK-80000 US VA0 as recommended  I taped off pin 7 on the small board and all the problems seem to be gone!

**Comment by rayman-g** [comment]
> > Yeah I did the tape trick on the CD drive pin in the end and the Saroo is working fine now with no problems. Just wondered if there was a solution that means I can still use my CD drive if needed.
> 
> If only the B7 PIN is processed, the CD drive will not be affected. You can still use CD

I have exact issue with sound glitches on va0, is there a video turorial or pictures for taping B7 PIN or schematics?
no soldering things.

**Comment by Backloggaming** [comment]
Is there a chance that the sound glitch can be fixed by a new firmware update? I’m using version 1 Japanese Saturn. And I noticed a pattern while playing Asuka 120%. The sound glitches comes on and off depending on what state you are on. It seems affecting the SFX memory channel only and not the BGM.

**Comment by Backloggaming** [comment]
> > > Yeah I did the tape trick on the CD drive pin in the end and the Saroo is working fine now with no problems. Just wondered if there was a solution that means I can still use my CD drive if needed.
> > 
> > 
> > If only the B7 PIN is processed, the CD drive will not be affected. You can still use CD
> 
> I have exact issue with sound glitches on va0, is there a video turorial or pictures for taping B7 PIN or schematics? no soldering things.

Here you go… I documented my fix and it works 
https://youtu.be/nWIs6qGuU84?si=3QFqxoIHDHmdEmBO

**Comment by migueljerico** [comment]
I have a VA0 japanese Saturn which had plenty of audio errors: I had to tape B7 pin on SH-1 CD Rom Drive Board, turn sound to mono...anything helped, because I still had glitches.

But reading another thread here about how to configure Saroo to have default CD speed and more stability, I decided to write sector_delay=6000 and play_delay=30000 on GLOBAL settings via cfg, and voilà, this has eliminated all my problems with no soldering.

Now everything works OK without any glitch, also if I change music to stereo, no sound problems at all.

So I recommend everybody who has this problems to do that and I can now finally enjoy my Saroo.

**Comment by mdi82** [comment]
> I decided to write sector_delay=6000 and play_delay=30000 on GLOBAL settings via cfg, and voilà, this has eliminated all my problems with no soldering.
> 
> So I recommend everybody who has this problems to do that and I can now finally enjoy my Saroo.

I tried this and I still have audio glitches on mine. Guess I have to stick with tape.

**Comment by migueljerico** [comment]
> > I decided to write sector_delay=6000 and play_delay=30000 on GLOBAL settings via cfg, and voilà, this has eliminated all my problems with no soldering.
> > So I recommend everybody who has this problems to do that and I can now finally enjoy my Saroo.
> 
> I tried this and I still have audio glitches on mine. Guess I have to stick with tape.

Maybe is a combination of both, I still had glitches with only tape but now everything is OK.

Taping B7 pin is easy. Good luck.

**Comment by migueljerico** [comment]
Hi.

I updated to 0.9 and I had some glitchy sounds and graphics again, although having pin B7 taped.

So I investigated through the web and I found that there is a private config (I don't know why this is private) to reduce the fpga speed and also reduce SCPclk problems.

You only have to write fpga_speed=1 on GLOBAL settings via cfg, this seems that this adjusts the SCPclk, you are making the Saroo more tolerant of the signal noise specific to our VA0 board.

I did not read anything like that in public I don't know whay, but it worked for me.
